lib/blueprints/component/temp.component.spec.extension

```diff
@@ -1,29 +0,0 @@
-import Vue from 'vue';
```

Comment:
Why remove the spec file?

Reply:
only rename. i remove 'component' from name